Source/ASDisplayNode.mm

```diff
@@ -2005,8 +2033,10 @@ + (void)scheduleNodeForRecursiveDisplay:(ASDisplayNode *)node
 }
 
 /// Helper method to summarize whether or not the node run through the display process
-- (BOOL)__implementsDisplay
+- (BOOL)_implementsDisplay
 {
+  ASDN::MutexLocker l(__instanceLock__);
```

Comment:
Yes agree also `layerBacked`. I think we should defer that to another PR as the diff is already big and `_flags` are used all over the place.